recipe/meta.yaml

```
@@ -1,17 +1,20 @@
 {% set name = "pythia8" %}
-{% set version = "8.240" %}
+{% set version = "8.2.44" %}
```

Actually this is a bad idea, I guess we can do something clever with the `max_pin` argument to `pin_subpackage`:
```suggestion
{% set version = "8.244" %}
```